src/main/java/com/neighborhoodAssurance/dao/LocationDAO.java

```diff
@@ -0,0 +1,4 @@
+package com.neighborhoodAssurance.dao;
+
+public class LocationDAO {
```

Comment:
This looks like a partially implemented feature.  I wouldn't add this until the feature is fully implemented.  If this branch is merged, it will be merged into master, and only complete features should be merged to master.  Incomplete features should remain in the feature branch, until the feature branch meets the definition of done.